libs/logging/src/main/java/org/elasticsearch/logging/LogManager.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+package org.elasticsearch.logging;
+
+import org.elasticsearch.logging.internal.spi.LogManagerFactory;
+
+/**
+ * A class used for creating loggers.
+ */
+public final class LogManager {
+
+    /**
+     * Returns a Logger with the specified name.
+     *
+     * @param name The logger name.
+     * @return The Logger.
+     */
+    public static Logger getLogger(final String name) {
+        return LogManagerFactory.provider().getLogger(name);
```

Comment:
I assume null in results in a NPE? If so, maybe a class-level comment, e.g.

  `Unless otherwise noted, passing a null argument to a method of any class will cause a NullPointerException`

Reply:
actually this depends on log4j. and if a null is passed as a `name` or `clazz` parameter, it will take caler's class 
`StackLocatorUtil.getCallerClass(2)`

Reply:
We should clarify this in the javadoc, no ? Or alternatively, just disallow null (throw NPE)

Reply:
Has this been address in the javadoc? I'm not seeing it

Reply:
I did not add the javadoc. This depends on how log4j does this and I did not want to mention this in the API's javadoc. I think is is rare that a name is null.